packages/components/inputs/selectable-search-input/src/selectable-search-input.tsx

```diff
@@ -424,7 +424,7 @@ const SelectableSearchInput = (props: TSelectableSearchInputProps) => {
       }
       textInputRef.current?.focus();
     },
-    [onChange, selectablSearchInputId, name]
+    [props, selectablSearchInputId, name]
```

Comment:
this will make the callback to be regenerated on every props updates. Perhaps we can use the `props.onChange` property in the array so it gets recreated only if that prop in new.

Reply:
Updated here: https://github.com/commercetools/ui-kit/pull/2776/commits/12a589e2aef12601ea4cd772895d4b79145d2bc7 and here: https://github.com/commercetools/ui-kit/pull/2776/commits/12d690966cbe724414dbd466a44c1191f082e583

Thank you.